src/app/assets/languages/en.json

```diff
@@ -210,7 +210,7 @@
           "cache-feature-list-placeholder": "Select capabilities",
           "cache-feature-list-typeahead": "Select capabilities",
           "bounded": "Bounded",
-          "bounded-tooltip": "Control the size of the cache by configuring {{brandname}} to evict entries. You can configure an eviction threshold based on the maximum amount of memory that a cache can use or based on the total number of entries that a cache can hold.",
+          "bounded-tooltip": "To limit the size of the cache, configure {{brandname}} to evict entries. You can set an eviction threshold based on either the cache’s maximum amount of memory or based on the total number of entries that it can hold.",
```

Comment:
Could say "to restrict" instead of "to limit". Also we've never used possessive with the word cache to avoid anthropomorphism. I think saying "based on the maximum amount of memory or total number of entries" is fine.